cf/mixin/propertiesdata.py

```diff
@@ -3457,6 +3457,36 @@ def file_locations(self):
 
         return set()
 
+    @_inplace_enabled(default=False)
+    def filled(self, fill_value=None, inplace=False):
+        """Replace masked elements with a fill value.
+
+        .. versionadded:: NEXTVERSION
+
+        :Parameters:
+
+            fill_value: scalar, optional
+                The fill value. By default the fill returned by
+                `get_fill_value` is used, or if this is not set then
+                the netCDF default fill value for the data type is
+                used (as defined by `netCDF.fillvals`).
```

Comment:
Not sure what specifically this refers to (a `netCDF` class here or in cfdm?), from a quick poke around we could use either:

* our `cf.default_netCDF_fillvals()` method; or
* `netCDF4.default_fillvals` as the under-the-hood origin;
* 
either of which might be more appropriate - certainly they would be more searchable from the docs to the end user.

I suggest the first is best but up to you...

```suggestion
                used (as defined by `cf.default_netCDF_fillvals`).
```